tests/bluetooth/init/prj_ctlr_peripheral_priv.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_BT=y
+CONFIG_BT_CTLR=y
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_OBSERVER=n
+CONFIG_BT_CENTRAL=n
+CONFIG_BT_SMP=y
+CONFIG_BT_PRIVACY=y
+CONFIG_BT_EXT_ADV=n
+CONFIG_BT_CTLR_ADV_EXT=n
+CONFIG_BT_LL_SW_SPLIT=y
+CONFIG_ZTEST=y
```

Comment:
These files need entries in testcase.yaml to get included in CI

Reply:
Updated testcase.yaml, thank you.